core/src/main/scala/cromwell/core/callcaching/CallCachingMode.scala

```diff
@@ -13,6 +13,8 @@ sealed trait CallCachingMode {
 }
 
 case object CallCachingOff extends CallCachingMode {
+  override val readFromCache = false
+  override val writeToCache = false
```

Comment:
I'm hearing a lot of points...but I'm leaving it for now but would have no hurt feelings if someone in a future PR got rid of it :)